src/style.css

```diff
@@ -1,3 +1,73 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.header {
+  margin: 0;
+  display: flex;
+  justify-content: space-between;
+  /* background-color: rgb(118, 118, 118); */
```

Comment:
Remove  all comments. Try it write obvious code that explain all we need without comments)